packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -22,7 +22,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
   // in Jest configs. We need help from somebody with Windows to determine this.
   const config = {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
-    setupFiles: [resolve('config/polyfills.js')],
+    setupFiles: [resolve('config/jest/polyfills.js')],
```

Comment:
Why not `react-app-polyfill/jsdom` directly

Reply:
Good point. Though, this might be a good default for people who eject so they have a location to easily add new polyfills required for tests.

Reply:
And if we do switch this, does it need to be `<rootDir>/node_modules/react-app-polyfill/jsdom`?

Reply:
>Good point. Though, this might be a good default for people who eject so they have a location to easily add new polyfills required for tests.

At this point they might as well change the config directly.

>And if we do switch this, does it need to be <rootDir>/node_modules/react-app-polyfill/jsdom?

Check Jest docs? I would expect it to be smart enough to resolve it somehow.